bindings/ios/MEGASdk.h

```diff
@@ -7186,6 +7187,38 @@ typedef NS_ENUM(NSInteger, BusinessStatus) {
  */
 - (void)getRegisteredContacts:(NSArray<NSDictionary *> *)contacts delegate:(id<MEGARequestDelegate>)delegate;
 
+#pragma mark - Push Notification Settings
+
+/**
+ *@brief: Get push notification settings
+ *
+ * The associated request type with this request is MEGARequestTypeSetAttrUser
+ * - [MEGARequest paramType] - Returns the attribute type MEGAUserAttributePushSettings
+ *
+ * Valid data in the MEGARequest object received in onRequestFinish when the error code
+ * is MEGAErrorTypeApiOk:
+ * - [MEGARequest megaPushNotificationSettings] Returns settings for push notifications
+ *
+ *@param delegate MEGARequestDelegate to track this request
+ */
+- (void)getPushNotificationSettingsWithDelegate:(id<MEGARequestDelegate>)delegate;
```

Comment:
We need to add the method to get the push notifications without the needed of a delegate. 

`- (void)getPushNotification`